packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -294,20 +294,18 @@ export class MessageWidget extends Component {
     const {props} = this;
     const handleEvent = this.handleEvent;
     const toUser = space.toPerson;
-    props.updateMercuryState({isListeningToActivity: true});
+    props.updateWidgetState({isListeningToActivity: true});
     sparkInstance.internal.mercury.on(`event:conversation.activity`, (event) => {
       const activity = event.data.activity;
       const isSelf = activity.actor.id === props.user.get(`currentUser`).id;
       // Ignore activity from other conversations
       if (activity.target && activity.target.id === space.id) {
         if (activity.object.objectType === `activity`) {
-          props.receiveMercuryActivity(activity);
           if (isSelf && activity.verb === `acknowledge`) {
```

Comment:
Can we combine the 2 sequential if statements into a single one?

Reply:
Oh yea, will do.